src/ServiceControl/Operations/Msmq/MsmqAuditQueueImporter.cs

```diff
@@ -27,12 +27,6 @@ public MsmqAuditQueueImporter(IDocumentStore store, IBuilder builder, IDequeueMe
             this.store = store;
             this.builder = builder;
             enabled = receiver is MsmqDequeueStrategy;
```

Comment:
Agreed, lets touch base with @johnsimons and treat that as a seprate task if needed.

I shouldn't have brought it up here. My bad!